common/config.py

```diff
@@ -397,7 +379,7 @@ def checkConfig(self):
                         self.notifyError(
                             '{}\n{}'.format(
                                 _('Profile: "{name}"').format(
-                                    name=self.currentProfile()),
+                                    name=profile_name),
```

Comment:
Good change to use the name instead of the ID!

I personally even love to see not only the name  but also the ID (eg. "Main Profile [1]")
which helps eg. in screen shots in issues...

Reply:
Good idea. But can we keep this in a separate issue?
There are a lot of code locations where the profile name/id is shown.

Reply:
Yes of course, no change required (we could use a bigger GUI refactoring for this)